code/_onclick/hud/swarmer.dm

```diff
@@ -13,6 +13,22 @@
 		var/mob/living/simple_animal/hostile/swarmer/S = usr
 		S.CreateTrap()
 
+/**
+  * # SwarmerPylonButton
+  *
+  * The ui object operating as a button to create pylons.
+  */
+
+/obj/screen/swarmer/Pylon
+	icon_state = "ui_barricade"
+	name = "Create pylon (Costs 5 Resources)"
+	desc = "Creates a pylon, which expands the area swarmers may operate in. (Costs 25 resources)"
```

Comment:
So, is this intended to be 25 or 5? Further down it's set to 25 so the name could just be a typo.

Reply:
My intention was 25, typing 5 was a mistake.

This value was chosen via guesstimate.  I hold no attachment to this value whatsoever.